src/Tizen.Data.Tdbc/Tizen.Data.Tdbc.Driver.Sqlite/Statement.cs

```diff
@@ -0,0 +1,175 @@
+﻿/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace Tizen.Data.Tdbc.Driver.Sqlite
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class Statement : IStatement
+    {
+        Connection _conn;
+        private IntPtr _stmt;
+        private bool disposedValue;
+
+        private void Clear()
+        {
+            if (_stmt != IntPtr.Zero)
+            {
+                Interop.Sqlite.Finalize(_stmt);
+                _stmt = IntPtr.Zero;
+            }
+        }
+
+        private bool Prepare(Sql sql)
+        {
+            Clear();
+            Console.WriteLine("Prepare: " + sql.Command);
+            int ret = Interop.Sqlite.Prepare(_conn.GetHandle(), sql.Command, -1, out _stmt, IntPtr.Zero);
+            if (ret != (int)Interop.Sqlite.ResultCode.SQLITE_OK)
+            {
+                Console.WriteLine("Prepare: failed " + ret);
+                return false;
+            }
+
+            foreach (var i in sql.Bindings)
```

Comment:
does it handle null values?

Reply:
sqlite can binds null type value, I will prepare null type binding later.